docs/site/Server.md

```diff
@@ -69,40 +69,6 @@ const app = new RestApplication({
 });
 ```
 
-### Customize How OpenAPI Spec is Served
```

Comment:
In order to make it easier to find related guides when reading "Server" explanation page, perhaps we should add a new section here with links to those guides. I am thinking about something along the following lines:

```markdown
## Common tasks

- [Customize how OpenAPI spec is served](rest-customize-openapi-serving.html)
- [Enable HTTPS](deployment-guides-enabling-https.html)
```